semeio/workflows/misfit_preprocessor/job.py

```diff
@@ -18,7 +17,7 @@ def run(config, measured_data, reporter):
     scaling_configs = spearman_job(
         measured_data,
         reporter,
-        cluster_function=workflow.clustering.type,
+        cluster_function=workflow.clustering._cluster_function,
```

Comment:
Not entirely sure how I feel about this, but think I might like it, it keeps what is being configured close to the configuration. But it could be over complicating things.

Reply:
I don't think it overcomplicates things by passing the function as part of the configuration. Better than having some sort of mapping somewhere  which maps string to func. I think we should be consistent though, and if there are other places which would map to a function, we could likely have those places use the same behaviour.